pom.xml

```diff
@@ -16,6 +15,15 @@
         <module>bookshelf</module>
     </modules>
 
+    <reporting>
```

Comment:
@shun-fan @tswast Would it make sense to put this in the parent pom, to keep it out of the way?